test/app-luatest/gh_5444_relax_uuid_validation_test.lua

```diff
@@ -0,0 +1,48 @@
+local uuid = require('uuid')
+local msgpack = require('msgpack')
+local t = require('luatest')
+
+local g = t.group()
+
+-- Base test.
```

Comment:
missed dot at the end of comment :-/

Reply:
Do you mean the 7th line or [the 11th](https://github.com/tarantool/tarantool/pull/10185/files/cc3eef22e3f767e9b03acce5de4686bd4fddd772..6b9f524b666ddfb7ea4a76ec983b1f0947f37492#diff-52f768d375974430f8111e732d95aa320c30c47f37b1c8760b9422c2864c69edR11)?
Should I add the dot at the end of the link?